src/Options.php

```diff
@@ -287,6 +287,18 @@ public function setInAppExcludedPaths(array $paths): void
         $this->options = $this->resolver->resolve($options);
     }
 
+    public function getInAppIncludedPaths(): array
```

Comment:
Missing the PHPDoc block

Reply:
To clarify, I would like the PHPDoc to document that the returned array is a list of `string`